src/Searchable.php

```diff
@@ -26,38 +27,59 @@
  */
 class Searchable extends DataExtension
 {
-    public static $published_field = 'SS_Published';
+    public const TYPE_FIELD = 'type';
+
+    public const PUBLISHED_FIELD = 'SS_Published';
```

Comment:
A terse descripton of what these constants are for would be useful + docstrings.

Reply:
Fixed